src/resolvers/Mutation/createPost.ts

```diff
@@ -81,6 +83,19 @@ export const createPost: MutationResolvers["createPost"] = async (
     }
   }
 
+  // Check title and pinpost
+  if (args.data?.title && !args.data.pinned) {
```

Comment:
1. Pinned posts can only be created by Super Admins or Admins for the organization where the post was created.
2. Please make sure this logic is enforced